### ISSUE TITLE
Table No Data text is not editable

### ISSUE THREAD

**Issue by kagankorkmaz** [eleme/element-react#1108]
### Description

When there is no data in the table the text will be displayed is not editable it does not exist on the TableProps.
Moreover I internationalized the element-react to English by using 
`
import { i18n } from "element-react";
import locale from "element-react/src/locale/lang/en";
i18n.use(locale);
`

And still the text displaying automatically is "暂无数据"

It is very annoying and I guess easily solvable. If you can help I will be so appreciated.

**Comment by kamxy** [comment]
same issue

**Comment by ceyhuncfc9** [comment]
Yeah I have this issue too, really frustrating actually. I didn't have this issue when I did the same configuration with the Vue.js package.

**Comment by AlexVagrant** [comment]
I think this link may help you: https://elemefe.github.io/element-react/index#/zh-CN/i18n
### webpack.config.js
```js
{
  plugins: [
    new webpack.NormalModuleReplacementPlugin(/element-react[\/\\]src[\/\\]locale[\/\\]lang[\/\\]zh-CN/, 'element-react/src/locale/lang/en')
  ]
}
```

**Comment by kagankorkmaz** [comment]
> I think this link may help you: https://elemefe.github.io/element-react/index#/zh-CN/i18n
> 
> ### webpack.config.js
> ```js
> {
>   plugins: [
>     new webpack.NormalModuleReplacementPlugin(/element-react[\/\\]src[\/\\]locale[\/\\]lang[\/\\]zh-CN/, 'element-react/src/locale/lang/en')
>   ]
> }
> ```

Hi Alex, thank you so much for your response. I solved that by copying repo and using it as a local npm module also by that way I solved some other issues that I gave up. They mostly forgot add typings, if you have similar problems with me I strongly suggest.
